lpcompiler.pas

```diff
@@ -1459,7 +1459,7 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
     end;
 
     try
-      if (Tokenizer.Tok = tk_kw_Overload) or FuncHeader.isOperator then
+      if (Tokenizer.Tok = tk_kw_Overload) or (FuncHeader.isOperator and (Tokenizer.Tok <> tk_kw_Override)) then
```

Comment:
There are two keywords parsed at this point it's `override`, and `overload`, if it's "isOperator" then `overload` is implied, unless `override` is explicitly declared.
So yes it will accept any keyword, like it did in my original pull-request, as overload is implied it should always enter there, unless...